src/MongoDB.Driver.Core/Core/Operations/MongoBulkWriteOperationException.cs

```diff
@@ -60,6 +60,13 @@ public class MongoBulkWriteOperationException : MongoServerException
             _writeErrors = writeErrors;
             _writeConcernError = writeConcernError;
             _unprocessedRequests = unprocessedRequests;
+            if (_writeConcernError != null)
+            {
+                foreach (var errorLabel in _writeConcernError.ErrorLabels)
```

Comment:
Note: I don't check `ErrorLabels` for `null` since there's `Ensure.IsNotNull` above. If `Ensure` is removed, `null` check should be added. Same for other occurrences.